plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/access/DBAuthUtils.java

```diff
@@ -84,4 +88,21 @@ public static boolean promptAndChangePasswordForCurrentUser(
         }
         return false;
     }
+
+    public static boolean isPageCompleteByAuthModel(
```

Comment:
Let's rename it to isHostPresent.
Btw port is optional for almost all database driver. Why we check it as required?